e2e-tests/insurance/cypress/e2e/journeys/policy/broker-details/broker-details-page.spec.js

```diff
@@ -162,11 +162,23 @@ context(
         cy.assertUrl(brokerConfirmAddressUrl);
       });
 
-      describe('when going back to the page', () => {
+      describe(`when submitting ${IS_BASED_IN_UK} as no and going back to the page`, () => {
```

Comment:
just to confirm, should this be as no or = no?

Reply:
Should be `as "no"`. I've updated various tests to be aligned.